applications/nrf5340_audio/src/audio/streamctrl_unicast_client.c

```diff
@@ -14,37 +14,19 @@
 #include "macros_common.h"
 #include "audio_system.h"
 #include "button_handler.h"
-#include "data_fifo.h"
 #include "le_audio.h"
 #include "bt_mgmt.h"
 #include "bt_rend.h"
 #include "audio_datapath.h"
```

Comment:
I believe this can be removed now

Reply:
Strange. This was removed on my side. Will check.

Reply:
Has not been done